docker/local/development/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     environment:
       - SERVICES=s3
     ports:
-      - "4566:4566"
+      - "${DOCKER_LOCALSTACK_PORT}:4566"
```

Comment:
```suggestion
      - "${DOCKER_LOCALSTACK_PORT:-4566}:4566"
```

Reply:
I am not a 100% sure if this works but it's a nice touch to add overridable, default values.